ubyssey/events/views.py

```diff
@@ -125,4 +150,4 @@ def get_host_from_url(url):
     if m:
         return m.group(1)
     else:
-        raise EventError('URL provided is not a valid Facebook event or UBC event url')
```

Comment:
Ah if we raise the error, the `submit_form` method will stop executing when trying to pull Facebook data. if we return `None`, it will turn `url_error` to `True` and the submit form displays properly